src/Truncate/README.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Truncate'
+type: 'component'
+components:
+- Truncate
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+---
+
+A Truncate component can help you crop multiline text. There will be three dots at the end of the text.
+
+### Basic Usage
+
+```jsx live
+  <Truncate lines={2}>
+    Learners, course teams, researchers, developers: the edX community includes groups with a range of reasons 
+    for using the platform and objectives to accomplish. To help members of each group learn about what edX 
+    offers, reach goals, and solve problems, edX provides a variety of information resources.
+    Learners, course teams, researchers, developers: the edX community includes groups with a range of reasons
+    for using the platform and objectives to accomplish. To help members of each group learn about what edX
+    offers, reach goals, and solve problems, edX provides a variety of information resources.
+  </Truncate>
+```
+
+### With the custom ellipsis
+
+```jsx live
+() => {
```

Comment:
Let's keep this example solely about the custom ellipsis, and add a separate example for using `onTruncate`. For now, we can just include an example that includes a `console.log()` inside of `onTruncate`.

Reply:
Good, done